test.js

```diff
@@ -0,0 +1,15 @@
+var request = require('request');
+
+request.post({
+    url: 'http://localhost:8000/metadata',
+    method: 'POST',
+    json: { test: "test" }
```

Comment:
Does this work?

I'm used to seeing:

```javascript
json: true,
body: { test: "test" }
```

If it works, I think your version is preferable.